DSTClusterConfig/mods/README.md

```diff
@@ -28,7 +28,8 @@ Here's what I do to "generate" the `configuration_options` the _easy_ way:
 1. Skip world config, focus on the "Mods" tab: enable & configure the server mods the way you want (you need to have "Subscribed" to the mods in the Steam Workshop for them to appear there)
 1. Hit "Generate World". Do actually create a game. Once the world was created, you may exit the game.
 1. Now a `modoverrides.lua` was generated for you based on the settings you chose! You may find it in:
-    * Mac/Linux: `~/.klei/DoNotStarveTogether/Cluster_{N}/Master/modoverrides.lua`
+    * Linux: `~/.klei/DoNotStarveTogether/Cluster_{N}/Master/modoverrides.lua`
+    * Mac: `~/Documents/Klei/DoNotStarveTogether/{Some Numbers}/Cluster_{N}/Master/modoverrides.lua`
```

Comment:
```suggestion
    * Mac: `~/Documents/Klei/DoNotStarveTogether/{SteamAccountID}/Cluster_{N}/Master/modoverrides.lua`
```

I checked and the number seems to be the user's Steam Account ID. That's probably to keep data distinct if different users logs into Steam on the same computer 🤓

Reply:
Yes, it is. I checked mine on [steamidfinder.com](steamidfinder.com) and the number matched "steamID3" field.